### DISCUSSION TITLE
How to bind a control property to a member of a dynamic type?

### DISCUSSION THREAD

**Discussion by TheRealQuantam** [AvaloniaUI/Avalonia#17177] [Q&A]
In Avalonia 0.10 (I'm dealing with an OSS project that hasn't yet upgraded to 11) is it possible to bind to a member of either a dynamic type that is not fully described by reflection, either `dynamic` or something like `ExpandoObject`? E.g. to bind to `A.B` where `A` is one of the aforementioned?

**Comment by maxkatz6** [comment]
You should be able to implement custom `IPropertyAccessorPlugin` and install it there https://github.com/AvaloniaUI/Avalonia/blob/release/0.10.11/src/Avalonia.Base/Data/Core/ExpressionObserver.cs#L21

You probably wouldn't be able to find any examples of that though, as it's pretty uncommon scenario.

**Reply by maxkatz6** [reply]
Although I found this https://github.com/AvaloniaUI/Avalonia/discussions/16806
